app/models/sessionData/SessionCookieData.scala

```diff
@@ -35,8 +35,7 @@ case class SessionCookieData(mtditid: String,
       SessionKeys.clientMTDID -> mtditid,
       SessionKeys.clientNino -> nino,
       SessionKeys.clientUTR -> utr,
-      SessionKeys.isSupportingAgent -> isSupportingAgent.toString
-    ) ++ clientFirstName.map(SessionKeys.clientFirstName -> _) ++ clientLastName.map(SessionKeys.clientLastName -> _)
+      SessionKeys.isSupportingAgent -> isSupportingAgent.toString) ++ clientFirstName.map(SessionKeys.clientFirstName -> _) ++ clientLastName.map(SessionKeys.clientLastName -> _)
```

Comment:
I would assume we can revert this file as I would expect this is just a formatting ?